lib/tasks/biran_tasks.rake

```diff
@@ -7,12 +7,32 @@ namespace :config do
     Rake::Task['config:generate_with_deps'].invoke
   end
 
+  task :generate2 do
```

Comment:
This method is for discussion. Running this with multiple errors will generate output similar to the following:
```
Warning: Missing required argument or bad formatting in config file for sidekiq
Warning: Missing template file for sidekiq2: /Users/brian/repos/bs/config/_sidekiq2.erb
Errors in creating config files
brianl-macbook-pro-8: brian 12:53:54 [bs] test_biran_local [1] bs $ echo $?
1
```
You can see it attempted all of them, however the error code is still not 0, which means the task failed. I had to catch that on each task by looping through each task this way. I can look at the original task_with_deps approach again, but the errors were happening in the dependency tasks with no "wrapper" to catch them. 

We don't have to go down this route, I just found it interesting to explore while I was in here and testing.